22.md

```diff
@@ -0,0 +1,88 @@
+NIP-22
+======
+
+Comment
+-------
+
+`draft` `optional`
+
+A comment is meant to be used to comment on
+or reply to anything except event kinds
+whose specifications have already defined other
+ways to comment on or reply to them.
+
+For example, it can be used to comment on a [NIP-23](23.md) `kind:30023` nostr blog post
+or on an off-nostr news article or to reply to another comment.
+
+## Structure
+
+A comment is an event of `kind:1111` with plaintext `.content`
+(no HTML, no Markdown nor other formatting).
+
+It must have a `k` tag pointing to the kind of the subject being commented on.
+For example, when commenting on a `kind:30023`, the `k` tag is set to "30023".
+When replying to another `kind:1111`, the `k` tag is set to "1111".
+If commenting on an `https://abc.com/articles/1` artcile the `k` tag is set to "https://abc.com" (its domain).
+
+### Threads
+
+There is always an `o` tag pointing to the original post that started the thread.
+It can be an event id (with an `"i:<event-pubkey>:"` prefix),
+an event address (`"a:"` prefix) or an url (`"<protocol>"` prefix). The second value
+is the optional recommended relay url where the referenced event may be found.
+This tag is useful to load all messages of a thread at once.
+
+Note that if commenting on a replaceable event, one can choose to add both id and address references.
+
+Examples:
+
+- `["o", "i:<event-pubkey>:<event-id>", "<relay-url, optional>"]`
```

Comment:
Why is `<event-pubkey>` needed?

Reply:
Also decided to add kind to the reference. It is useful to let client know if it supports the referenced event before downloading it.